packages/portal/.env.example

```diff
@@ -178,6 +178,9 @@ OAUTH_CLIENT="YOUR_CLIENT"
 # JIRA_API_SERVICE_DESK_GALLERIES_CUSTOM_FIELD_SET_ID='customfield_10841'
 # JIRA_API_SERVICE_DESK_GALLERIES_CUSTOM_FIELD_SET_CREATOR_NICKNAME='customfield_10842'
 
+# Comma-separated list of hosts from which to accept feedback
+# FEEDBACK_ALLOWED_HOSTS=www.europeana.eu, pro.europeana.eu, test.europeana.eu, localhost:3000
```

Comment:
Remove the spaces between comma and host as it will add the space to the string resulting in values like ` localhost:3000` and mess up the validation.